integration/update_array_compat_test.go

```diff
@@ -238,3 +238,60 @@ func TestUpdateArrayCompatPullAll(t *testing.T) {
 
 	testUpdateCompat(t, testCases)
 }
+
+func TestUpdateArrayCompatAddToSetEach(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]updateCompatTestCase{
+		"Document": {
+			update: bson.D{{"$addToSet", bson.D{{"v", bson.D{
+				{"$each", bson.A{bson.D{{"field", int32(42)}}}},
+			}}}}},
+			skipForTigris: "Mixed types are not supported for Tigris.",
+		},
+		"String": {
+			update:        bson.D{{"$addToSet", bson.D{{"v", bson.D{{"$each", bson.A{"foo"}}}}}}},
+			skipForTigris: "Mixed types are not supported for Tigris.",
```

Comment:
This should work for tigris, so no need to skip right? We can have at least one test that covers tigris.

Reply:
Yes, the strings provider is working. Unskipped this test case.